### PR TITLE
MARP-994: Market-website-Artifact-and-version-in-download-feature-not save-update

### PR DESCRIPTION
**Pull request by linhpd-axonivy**

### CODE REVIEW

File: marketplace-ui/src/app/shared/components/loading-spinner/loading-spinner.component.html
Comment (nntthuy-axonivy) [inline marketplace-ui/src/app/shared/components/loading-spinner/loading-spinner.component.html:1]: do not separate conditions like this, use if-else instead: isFixPosition ? 'position-fixed' : 'position-absolute'

File: marketplace-ui/src/app/shared/components/loading-spinner/loading-spinner.component.scss
Comment (nntthuy-axonivy) [inline marketplace-ui/src/app/shared/components/loading-spinner/loading-spinner.component.scss:1]: bootstrap has position-fixed and position-absolute, you don't need to customize css

File: marketplace-ui/src/app/modules/product/product-detail/product-detail-version-action/product-detail-version-action.component.spec.ts
Comment (nntthuy-axonivy) [inline marketplace-ui/src/app/modules/product/product-detail/product-detail-version-action/product-detail-version-action.component.spec.ts:1]: some lines are not formatted, please check it

File: marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts [hunk 1]
```diff
@@ -146,6 +145,7 @@ export class ProductDetailComponent {
         this.productModuleContent.set(productDetail.productModuleContent);
         this.metaProductJsonUrl = productDetail.metaProductJsonUrl;
         this.productDetailService.productNames.set(productDetail.names);
+        localStorage.removeItem(STORAGE_ITEM);
```
Comment (nntthuy-axonivy) [inline marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts:148]: remove this line

Comment (linhpd-axonivy) [reply marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts]: done

**Review by nntthuy-axonivy — APPROVED** [review]
LGTM

**Pull request merged**